BPA/Ch.06/6.2.4_Model_Mh.R

```diff
@@ -12,27 +12,25 @@ set.seed(1234)
 ## The data generation code is in bpa-code.txt, available at
 ## http://www.vogelwarte.ch/de/projekte/publikationen/bpa/complete-code-and-data-files-of-the-book.html
 stan_data <- read_rdump("Mh.data.R")
-
 ## Initial values
 inits <- function() list(mean_p = 0.5, sigma = runif(1, 0.5, 0.9))
 
 ## Parameters monitored
 params <- c("N", "mean_p", "sigma", "omega")
 
 ## MCMC settings
-ni <- 15000
-nt <- 10
-nb <- 5000
+ni <- 10000
+nt <- 9
+nb <- 1000
 nc <- 4
 
 ## Call Stan from R
 out <- stan("Mh.stan",
             data = stan_data, init = inits, pars = params,
             chains = nc, iter = ni, warmup = nb, thin = nt,
             seed = 1,
-            control = list(adapt_delta = 0.95),
+            control = list(adapt_delta = 0.8),
```

Comment:
`adapt_delta = 0.8` is the default, so you can just skip it here and in future uses.